src/index.js

```diff
@@ -1,5 +1,6 @@
 import './style.css';
 import '@fortawesome/free-solid-svg-icons';
+
 const todo = ['go to gym', 'Study', 'work'];
```

Comment:
- [ ] Kindly make sure your task is an object containing three keys - description \[`string`\], completed \[`boolean`\], index: \[`number`\].
 
  _**Example:**_

            {
                description: 'An example described in the string',
                completed: false,
                index: 10,
            }

Reply:
hello @levy002  thank you for your comment should I name it index? or can I name it Id cuz id makes more sense to me